src/Schema/Events/Tap.ts

```diff
@@ -1176,3 +1176,28 @@ export interface TappedCompleteYourProfile {
   context_screen_owner_slug?: string
   user_id: string
 }
+
+/**
+ * A user taps to request a chage of payment method for the order/orders
+ *
+ * This schema describes event sent to Segment from [[tappedChangePaymentMethod]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedChangePaymentMethod",
+ *    context_module : "ordersHistory" | "paymentFailed"
```

Comment:
and in here we are capturing if the button was in paymentFailed banner or on the `ordersHistory` page. Is that understandable distinction? Proper way to encode those options?